packages/neuron-ui/src/widgets/TextField/index.tsx

```diff
@@ -30,7 +30,7 @@ const TextField = React.forwardRef(
       ...rest
     }: {
       field: string
-      label?: string
+      label?: string | React.ReactNode
```

Comment:
Perhaps it could be implemented as `label?: string | { node: ReactNode; text: string }`

Reply:
> Perhaps it could be implemented as `label?: string | { node: ReactNode; text: string }`

I think `label` should be strictly limited to `string`, so the definition will be recovered to `string`, and if a pre-component is necessary, an independent field as `prefix/suffix/preTab/postTab` can be added